CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [#315](https://github.com/alexa-js/alexa-app/pull/315): Fix card tests and required card type - [@kobim](https://github.com/kobim).
 * [#314](https://github.com/alexa-js/alexa-app/pull/314): Fix handling of `undefined` slot values - [@User1m](https://github.com/user1m).
+* Included Skill Builder example in example.ejs - [@funkydan2](https://github.com/funkydan2)
```

Comment:
Can you please make it look like the other changes here? Include the PR number and a period in the end.